src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -10,120 +10,122 @@ The following example PET datasets have been formatted using this specification
 and can be used for practical guidance when curating a new dataset.
 
 -   One subject single dynamic scan (pet, mri, blood):
-	[`pet_example1`](https://www.dropbox.com/sh/dmjmaxfswatptpm/AACw31cTyChLA1kwM4Vhqueja?dl=0)
+    [`pet_example1`](https://www.dropbox.com/sh/dmjmaxfswatptpm/AACw31cTyChLA1kwM4Vhqueja?dl=0)
+
 -   One subject single dynamic scan (pet, mri):
-	[`pet_example2`](https://www.dropbox.com/sh/i95a71rpq2unks7/AACby5XcYQp9vcwgx3cNXBhza?dl=0)
+    [`pet_example2`](https://www.dropbox.com/sh/i95a71rpq2unks7/AACby5XcYQp9vcwgx3cNXBhza?dl=0)
+
 -   One subject single dynamic scan (pet, blood):
-	[`pet_example3`](https://www.dropbox.com/sh/i0c3iq50id4s0d3/AABSjaczHM1SQnMRvh3fc7fna?dl=0)
+    [`pet_example3`](https://www.dropbox.com/sh/i0c3iq50id4s0d3/AABSjaczHM1SQnMRvh3fc7fna?dl=0)
 
 Further PET datasets are available from [OpenNeuro](https://openneuro.org).
 
-## PET recording data
-
-As specified above, this extension is relevant for brain PET imaging
-and its associated data, such as blood data.
-In addition, the extension is in accordance with the guidelines for reporting
-and sharing brain PET data (Knudsen et al. 2020, [doi:10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433)).
-
-Template:
+## Terminology and conventions
+
+PET-BIDS is fully consistent with the BIDS specification as a whole.
+However, BIDS was initially developed in the context of MRI,
+so some terminology may be unfamiliar to researchers from each field.
+This section adds clarifications to
+[Common Principles - Definitions](../02-common-principles.md#definitions)
+for the PET context, and introduces the term "time zero" which is currently
+specific to PET.
+
+1.  **Session** - In most cases, a new session with respect to PET corresponds
+    to a visit to the scanning site, and starts with a new injection.
+    In situations where different data types are obtained over several visits
+    (for example, FDG PET on one day followed by amyloid PET a couple days after)
+    these scans may be grouped into the same session.
+    In other datasets, a subject leaving the scanner and returning under the same
+    injection may be logically considered separate sessions.
```

Comment:
```suggestion
    injection may in some cases be considered separate sessions.
```

Reply:
I meant "logically" in terms of "for the purposes of analysis" (similar to a "logical partition" instead of "physical partition" in disk drives). I think I would prefer to just drop "logically".